app/javascript/components/AppHeader/index.tsx

```diff
@@ -22,6 +53,8 @@ export default function AppHeader() {
           <Button isLink href="/submit">
             Submit a paper
           </Button>
+          <ChangePasswordButton {...props}/>
+          <LogoutButton {...props}/>
```

Comment:
Here is a typical place where you'd decide whether to render these components or not:
```suggestion
          {loggedIn && (
            <>
              <ChangePasswordButton authenticityToken={props.authenticityToken} />
              <LogoutButton authenticityToken={props.authenticityToken} />
            </>
          )}
```